frontend-react/src/content/developer-resources/landing-page.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Developer Resources
+subtitle: Technical documentation for establishing a connection and learning more about how ReportStream works. 
+cards:
+    - title: API Guide
+      href: /api-guide
+      content: Learn about our API and how to set up your connection for reporting data to public health entities.
+    - title: GitHub
+      content: As an <a href="https://open.cdc.gov/">open source</a> project, all of our documentation is available on GitHub.
+    - title: Release notes
+      href: /release-notes
+      content: View monthly updates on new features, enhancements, and fixes.
+---
+<HeroWrapper>
+<section className="usa-section grid-container">
+<CardGroup>
+        <Card gridLayout={{tablet: {col: 4}}}>
+            <CardHeader>
+                ## [API Guide](/api-guide)
```

Comment:
This should be "ReportStream API" instead of "API Guide"

Reply:
What you're saying does not reflect the current figma sheet.